src/AtomsGaffer/AtomsCrowdReader.cpp

```diff
@@ -775,9 +776,10 @@ IECore::ConstCompoundObjectPtr AtomsCrowdReader::computeAttributes( const SceneN
 
 void AtomsCrowdReader::hash( const Gaffer::ValuePlug *output, const Gaffer::Context *context, IECore::MurmurHash &h ) const
 {
+    ObjectSource::hash( output, context, h );
```

Comment:
John H: `AtomsCrowdReader::hash()` is calling the base class hash at the wrong point. It should be calling it at the start of the function.